op-chain-ops/cmd/celo-migrate/README.md

```diff
@@ -82,11 +90,9 @@ go run ./cmd/celo-migrate full \
 --new-db ./data/alfajores_new
 ```
 
-The first time you run the script it should take ~5 minutes. The first part of the script will migrate ancient blocks, and will take the majority of the time.
-
-During the ancients migration you can play around with stopping and re-running the script, which should always resume where it left off. If you run the script subsequent times after ancient migrations have been run, the script should skip ancient migrations and proceed to migrating non-ancient blocks quickly.
+Running the full migration script re-runs the pre-migration script once to migrate any new changes to the old db that have occurred since the last pre-migration. It then performs in-place transformations on the non-ancient blocks and performs the state migration as well.
 
-Note that partial migration progress beyond the ancient blocks (i.e. non-frozen blocks and state changes) will not be preserved between runs by default.
+Once the full migration command has been run on a new db, re-running the migration script will require passing the `--clear-non-ancients` flag in order to reset all the non-ancient and state transformations that are made during a full migration.
```

Comment:
Maybe we could clean-up the commands and flags a bit and just have 3 commands:
 - pre-migration - safe to run multiple times before the actual migration, doesn't remove anything, append-only kind of
 - migration - you run it once when it's time, internally calls pre-migration one last time
 - clear - removes everything
 
 I'm not sure we need to be super-granular and have different flavors of the clear flags. What do you think?

Reply:
I think we could go one better just pre-migration and migration, since everything resides in the chaindata dir people can just delete it, no need for us to offer directory deletion as a command.